C4SGSvc/src/main/java/org/c4sg/controller/UserProjectController.java

```diff
@@ -0,0 +1,44 @@
+package org.c4sg.controller;
+
+import org.c4sg.dto.ProjectDto;
+import org.c4sg.dto.UserProjectDto;
+import org.c4sg.exception.UserProjectException;
+import org.c4sg.service.UserProjectService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@CrossOrigin
+@RestController
+@RequestMapping("/api/userProjects")
```

Comment:
It's a good point. So do you suggest to put those endpoints inside projects' path ?